platform/engineering/internal-tools.md

```diff
@@ -34,18 +35,15 @@ You do not need to run the SOCKS proxy while you're developing unless you need a
 
 <hr>
 
-
-> These steps assume you're running on Linux or OSX. There are slightly different commands to connect to the proxy depending on whether or not you are connected to the VA network. You will need to run the SOCKS proxy on your local system whenever you need access to tools on the `vetsgov-internal` domain.
-
 ## Design rationale
 
-Some utilities, such as metrics dashboards, error reporting, and deployment tools, should not be made available to the public. While such a utility may lock functionality behind a user login system, these vary among implementations and may be vulnerable to exploits. Additionally, due to the difficulty of creating subdomains for vets.gov, it may be problematic to host some of these utilities when they do not support relative path installations.
+Some utilities, such as metrics dashboards, error reporting, and deployment tools, should not be made available to the public. While such a utility may lock functionality behind a user login system, these vary among implementations and may be vulnerable to exploits. 
 
-The preferred solution is to host these systems on an internal network that prevents public access. While the VA maintains an internal network, the vets.gov team operates within the AWS environment. Installation of a utility like this on the VA network would require additional authority to operate extensions, a server within the VA, and mapping new connections from AWS to the VA. Since these tools are for use exclusively by the vets.gov services installed within AWS, utilizing the VA's internal network is unnecessary, more complex, time-intensive, and likely to impact performance of the utility and application requiring it.
+The preferred solution is to host these systems on an internal network that prevents public access. While the VA maintains an internal network, the VSP team operates within the AWS environment. After extensive efforts to obtain a zone delegation, we are able to control the records for internal tools using the `*.vfs.va.gov` zone delegation and Route53.
 
-To address this issue, we've configured an internal DNS server which can be accessed via a SOCKS proxy. The SOCKS proxy is a server on your local system that tunnels HTTP and DNS traffic to a jumpbox on the vets.gov AWS network. Once connected, a developer will have access to the `*.vetsgov-internal` TLD and can use their browser to connect to tools such as Prometheus.
+The SOCKS proxy is a server on your local system that tunnels HTTP and DNS traffic to a jumpbox on the va.gov AWS network. Once connected, a developer will have access to the `*.vfs.va.gov` domain and can use their browser to connect to tools such as Prometheus, Jenkins, and Sentry.
```

Comment:
I know this is small, but we may want to add Grafana to this list of tools, so people know.

Reply:
Added in https://github.com/department-of-veterans-affairs/va.gov-team/pull/11950/commits/ab4a6e42fe80e3f78ad62543645d53b6415589ec